ups/prompt-prototype.table

```diff
@@ -4,5 +4,6 @@
 #   the "base" package.
 setupRequired(base)
 setupRequired(sconsUtils)
+setupRequired(daf_butler)    # Used by middleware_interface module.
```

Comment:
I don't think this is ever going to be an eups package, so I'm not sure the `ups` directory is needed at all.